components/style/build_gecko.rs

```diff
@@ -558,6 +558,8 @@ mod bindings {
             "nsresult",
             "Loader",
             "ServoStyleSheet",
+            "EffectCompositor_CascadeLevel",
```

Comment:
The bindings update patch in 19aea7ea78ca0cb26287812765febcee71a84175 in #15175 doesn't look like it was written by hand, but it could just be that the old bindgen happened to generate this automatically but the new bindgen didn't.